security/keycloak-oidc-client/src/test/java/io/quarkus/ts/openshift/security/keycloak/AbstractSecurityKeycloakOpenShiftIT.java

```diff
@@ -0,0 +1,57 @@
+package io.quarkus.ts.openshift.security.keycloak;
+
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.api.model.KubernetesList;
+import io.fabric8.kubernetes.client.utils.Serialization;
+import io.fabric8.openshift.api.model.DeploymentConfig;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.quarkus.ts.openshift.app.metadata.AppMetadata;
+import io.quarkus.ts.openshift.common.CustomizeApplicationDeployment;
+import io.quarkus.ts.openshift.common.injection.WithName;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
+public abstract class AbstractSecurityKeycloakOpenShiftIT extends AbstractSecurityKeycloakOpenShiftTest {
+
+    private static final List<String> PROPERTIES_TO_SET_AUTH_URL = Arrays.asList("QUARKUS_OIDC_AUTH_SERVER_URL",
+            "QUARKUS_OIDC_CLIENT_AUTH_SERVER_URL",
+            "QUARKUS_OIDC_CLIENT_NORMAL_USER_AUTH_SERVER_URL",
+            "QUARKUS_OIDC_CLIENT_ADMIN_USER_AUTH_SERVER_URL",
+            "QUARKUS_OIDC_CLIENT_JWT_SECRET_AUTH_SERVER_URL");
+
+    static String keycloakUrl;
+
+    // TODO this is pretty ugly, but I'm tired and can't think of a better way at the moment
```

Comment:
We have this comment / code on 6 places already :)

Please create issue to get this done better way (if possible)

Reply:
There was already an issue: https://github.com/quarkus-qe/quarkus-openshift-test-suite/issues/127
Also, this is the related PR to fix this issue: https://github.com/quarkus-qe/quarkus-openshift-test-suite/pull/179

Reply:
yup, realized that once I added my comments :)